install.sh

```diff
@@ -212,6 +221,9 @@ main() {
 			usage
 			exit 0
 			;;
+		--with-terraform)
+			METHOD=install_terraform
```

Comment:
Should we call this `with_terraform` too for consistency?

Reply:
Sure!